### ISSUE TITLE
AVT 2 & 3 - React MultiSelect and Selectable Tile  - can not use the keyboard only to select or deselect a checkbox.

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#3474]
## Environment
Windows 10 
FireFox Quantum 68.0
JAWS 18

macOS 10.15.1
Chrome Version 78.0.3904.108 (Official Build) (64-bit)
VoiceOver

## Description
1. Tab to the MultiSelect Tile
2. Press Space bar to select a Tile, JAWS announces the checkbox but it can not be checked. 
**Expected Result:** User should be able to check 1 or more checkboxes.

Note: keyboard navigation fails with and without the use of a screen reader.

**Comment by snidersd** [comment]
@dakahn I also tested this on macOS with VO & Chrome and the issue is present there as well. In addition, keyboard navigation also fails without a screen reader, so I've updated the title.
